getambassadorio_blc.py

```diff
@@ -54,17 +54,23 @@ def is_internal_domain(self, netloc: str) -> bool:
 
     def product_should_skip_link(self, link: Link) -> bool:
         links_to_skip = [
-            'http://verylargejavaservice:8080/',
-            'https://blog.getambassador.io/search?q=canary',
-            'https://app.datadoghq.com/apm/traces/',
-            'http://web-app.emojivoto/',
-            'http://web-app.emojivoto/leaderboard/',
-            'http://verylargejavaservice.default:8080/color',
+            'http://localhost:3000/',
+            'http://localhost:3000/color',
             'http://localhost:8080/',
             'http://localhost:8083/',
             'http://localhost:8083/leaderboard/',
             'http://verylargejavaservice.default:8080/',
+            'http://verylargejavaservice.default:8080/color',
+            'http://verylargejavaservice:8080/',
+            'http://web-app.emojivoto/',
+            'http://web-app.emojivoto/leaderboard/',
+            'https://app.datadoghq.com/apm/traces/',
+            'https://blog.getambassador.io/search?q=canary',
+            'https://kubernetes.default',
+            'https://martinfowler.com/articles/microservice-testing/#testing-progress-1',
+            'https://tanzu.vmware.com/kubernetes-grid',
             'https://www.getambassador.io/*/',
+            'https://www.ticketmaster.com/',
```

Comment:
I think is the same case for ticketmaster

Reply:
for ticketmaster we got a 403 response, but I think it could be from the [agent](https://github.com/datawire/getambassador.io-blc2/blob/a122747917c5a1f0dab02c9169dec3d8e01f9a23/blclib/checker.py#L149).

Reply:
if the agent is the problem we need to fix the agent, skip adding this link as an exception